src/pydvl/influence/torch/functional.py

```diff
@@ -424,13 +437,16 @@ def hessian(
         def flat_input_batch_loss_function(
             p: torch.Tensor, t_x: torch.Tensor, t_y: torch.Tensor
         ):
-            return blf(align_with_model(p, model), t_x, t_y)
+            return blf(align_structure(params, p), t_x, t_y)
```

Comment:
Took me a while to understand that `blf` means `batch_loss_function`. I think `batch_loss` would be much clearer than `blf` and not too long (unless this is again a strong pytorch convention or so)

Reply:
Renamed to `batch_loss`, done.